src/components/Input/Input.test.tsx

```diff
@@ -49,6 +52,12 @@ describe("Renders Input", () => {
     container.simulate("click");
     expect(clickHander.callCount).to.equal(1);
   });
+
+  it("Changing the value calls the onChange handler", () => {
+    container.find("input").simulate("change", { target: { value: "Hello" } });
+
+    expect(changeHandler.callCount).to.equal(1);
```

Comment:
Wouldn't this be 5 since each character calls the onChange function?

Reply:
No, it's a fun Enzyme thing.  When you call `simulate("change")`it doesn't actually put the value in the input (meaning that calling `input.value.text` or the equivalent doesn't do anything.  All it does is call the changeHandler.  If you want to update the value you have to set `wrapper.props(value)` directly

Anyway, it's another reason I dislike Enzyme.